examples/podinfo/cue.mod/pkg/timoni.sh/core/v1alpha1/image.cue

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
```

Comment:
+1 for versioning this

Reply:
Would it makes sense to use `package image`, so it gets imported as that?

```
import "timoni.sh/core/v1alpha1:image"
```
WDYT?

Reply:
Assuming the core module will contain security context, resources and other helpers, do we want one package per type as in one package per file? In Go I would keep them under the same package.

Reply:
Go doesn't allow multiple packages per directory. It's just a thought. We can also say that this a niche feature of CUE that might more confusing then it is useful, but I don't have a particular opinion. I did use this feature ones in a package that contained multiple somewhat miscellaneous helpers.
I guess a reason CUE has this feature is to avoid unnecessary subdirs.

Reply:
From reading [the docs ](https://cuelang.org/docs/concepts/packages/#:~:text=Each%20package%20is%20identified%20by,%3A%20)%20and%20its%20package%20name.&text=If%20the%20basename%20of%20the,the%20latter%20can%20be%20omitted), I get a sense that the idea is simply so that package name doesn't have to match directory name and that there can be multiple packages in a directory.
Actually, this makes a lot of sense in contrast to Go, because there you can have a package name that differs from dir name, but that is not explicit and you actually have to know. CUE just makes this case explicit and introduces a possibility of multiple packages also.